src/map/entities/battleentity.h

```diff
@@ -239,9 +239,9 @@ enum SLOTTYPE
 
 #define MAX_SLOTTYPE 18
 
-// CROSSBOW и GUN - это Piercing, разделение сделано из-за одинакового skilltype
-// для возможности различить эти орудия при экипировке и избавиться от ошибки
-// использования пуль с арбалетом и арбалетных стрел с огнестрельным оружием (только персонажи)
+// CROSSBOW and GUN are Piercing, the division is made due to the same skilltype
```

Comment:
This can go because skill type does not universally correlate to damage type - there exceptions in multiple weapon categories where the skill type remains the same but the damage type does not. The most obvious ones being some old items like Heavy Halberd and Vampiric Claws. The lack of ranged weapons with unusual types at time of comment doesn't preclude the possibility of them existing later, but the comment leads readers to think this will always be true.